### PR TITLE
[FLINK-24787][docs] Add more details of state latency tracking documentation

### PR DESCRIPTION
**Pull request by masteryhx**
## What is the purpose of the change

This pull request adding more details about state latency tracking in the metric documentation.

## Brief change log

 - Added "State Access Latency" in metric.md

## Verifying this change

This change is a trivial work about documentation without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no
## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

File: docs/content.zh/docs/ops/metrics.md [hunk 1]
```diff
@@ -1259,6 +1259,157 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
...
+    <tr>
+      <th class="text-left" style="width: 18%">Scope</th>
+      <th class="text-left" style="width: 26%">Metrics</th>
+      <th class="text-left" style="width: 48%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th rowspan="27"><strong>Job (only available on TaskManager)</strong></th>
```
Comment (Myasuka) [inline docs/content.zh/docs/ops/metrics.md:1275]: I think the scope should be `Task/Operator`.

**Review by Myasuka — COMMENTED** [review]
Thanks for creating this PR, please take a look at my comments.

File: docs/content.zh/docs/ops/metrics.md (hunk 1 above)
Comment (masteryhx) [reply docs/content.zh/docs/ops/metrics.md]: I agree. Just modified.

**Review by Myasuka — APPROVED** [review]
Thanks for the update. LGTM.

**Pull request merged**